docker/docker_client_test.go

```diff
@@ -104,156 +105,163 @@ func TestGetAuth(t *testing.T) {
-			expectedPassword: "mypass",
-		},
-		{
-			name:       "match none",
-			hostname:   "registry.example.org",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{"example.org": testAuthConfigData{"joe", "mypass"}}),
-		},
-		{
-			name:     "match docker.io",
-			hostname: "docker.io",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"example.org":     testAuthConfigData{"example", "org"},
-				"index.docker.io": testAuthConfigData{"index", "docker.io"},
-				"docker.io":       testAuthConfigData{"docker", "io"},
-			}),
-			expectedUsername: "docker",
-			expectedPassword: "io",
-		},
-		{
-			name:     "match docker.io normalized",
-			hostname: "docker.io",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"example.org":                testAuthConfigData{"bob", "pw"},
-				"https://index.docker.io/v1": testAuthConfigData{"alice", "wp"},
-			}),
-			expectedUsername: "alice",
-			expectedPassword: "wp",
-		},
-		{
-			name:     "normalize registry",
-			hostname: "https://docker.io/v1",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"docker.io":      testAuthConfigData{"user", "pw"},
-				"localhost:5000": testAuthConfigData{"joe", "pass"},
-			}),
-			expectedUsername: "user",
-			expectedPassword: "pw",
-		},
-		{
-			name:     "match localhost",
-			hostname: "http://localhost",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"docker.io":   testAuthConfigData{"user", "pw"},
-				"localhost":   testAuthConfigData{"joe", "pass"},
-				"example.com": testAuthConfigData{"alice", "pwd"},
-			}),
-			expectedUsername: "joe",
-			expectedPassword: "pass",
-		},
-		{
-			name:     "match ip",
-			hostname: "10.10.3.56:5000",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"10.10.30.45":     testAuthConfigData{"user", "pw"},
-				"localhost":       testAuthConfigData{"joe", "pass"},
-				"10.10.3.56":      testAuthConfigData{"alice", "pwd"},
-				"10.10.3.56:5000": testAuthConfigData{"me", "mine"},
-			}),
-			expectedUsername: "me",
-			expectedPassword: "mine",
-		},
-		{
-			name:     "match port",
-			hostname: "https://localhost:5000",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"https://127.0.0.1:5000": testAuthConfigData{"user", "pw"},
-				"http://localhost":       testAuthConfigData{"joe", "pass"},
-				"https://localhost:5001": testAuthConfigData{"alice", "pwd"},
-				"localhost:5000":         testAuthConfigData{"me", "mine"},
-			}),
-			expectedUsername: "me",
-			expectedPassword: "mine",
-		},
-		{
-			name:     "use system context",
-			hostname: "example.org",
-			authConfig: makeTestAuthConfig(testAuthConfigDataMap{
-				"example.org": testAuthConfigData{"user", "pw"},
-			}),
-			expectedUsername: "foo",
-			expectedPassword: "bar",
-			ctx: &types.SystemContext{
-				DockerAuthConfig: &types.DockerAuthConfig{
-					Username: "foo",
-					Password: "bar",
+	for _, configPath := range configPaths {
+		fmt.Println(configPath)
```

Comment:
This could be incorporated into the test case name perhaps. Or at least use `logrus.Debugf` so that it is silenced on successful runs.

Reply:
fixed